docker-compose.local.yml

```diff
@@ -1,9 +1,9 @@
-version: '3.1'
+version: "3.1"
```

Comment:
@peter-asztalos is there any difference between single and double quotes here or it is just a matter of choice?

Reply:
Probably there is some, but it doesn't matter, it was formatted by prettier before (why??) but I set the formatter to [Red Hat's vscode extension](https://marketplace.visualstudio.com/items?itemName=redhat.vscode-yaml) so it changed to double quote. I had troubles with prettier if it wasn't installed in the local `node_modules` and applied JS rules (I think), so I think changing was a good decision.